external-crates/move/move-compiler/src/diagnostics/mod.rs

```diff
@@ -439,6 +439,18 @@ impl WarningFilters {
             },
         }
     }
+
+    pub fn unused_warnings_filter() -> Self {
+        let mut filtered_codes = BTreeSet::new();
```

Comment:
nit, you should be able to use `BTreeSet::from([...])` and then you don't need the inserts